kalliope/core/NeuronModule.py

```diff
@@ -73,17 +73,23 @@ def __init__(self, **kwargs):
         brain_loader = BrainLoader()
         self.brain = brain_loader.brain
 
+        self.override_params = dict()
+        # get if the cache settings is present
+        override_cache = kwargs.get('cache', None)
+        if override_cache is not None:
+            self.override_params['cache'] = override_cache
+
         # check if the user has overrider the TTS
         tts = kwargs.get('tts', None)
         if tts is None:
             # No tts provided,  we load the default one
             self.tts = self.settings.default_tts_name
+        elif type(tts) is dict:
+            self.tts = tts.keys()[0] 
+            self.override_params = tts.values()[0]
```

Comment:
I still really don't like this method. hard coding a position is not a good practice